### PR TITLE
Refine headers for filter and total products

### PR DESCRIPTION
**Pull request by stsci-hack**
These changes define separate sets of rules for filter and total drizzle products from the rules used for single input exposure drizzle products.  This allows keywords like IPPPSSOO and ASN_ID which only apply to single exposures (or data from the same ASN) to be removed from SVM filter and total drizzle products and from MVM layers drizzle products.

### CODE REVIEW

**Comment by stsci-hack** [comment]
These changes were tested by processing data from SVM visit 'icde25'.

File: drizzlepac/haputils/processing_utils.py [hunk 1]
```diff
@@ -50,6 +50,13 @@ def get_rules_file(product, rules_type=""):
         exposure.  Valid values: blank/empty string (''), 'SVM' or 'svm'
         for SVM processing (default) and 'MVM' or 'mvm' for MVM processing.
 
+    rule_root : str, optional
```
Comment (mdlpstsci) [inline drizzlepac/haputils/processing_utils.py:53]: Just a typo - rule_root should be rules_root.

File: drizzlepac/pars/acs_header_hap.rules [hunk 2]
```diff
@@ -401,13 +401,30 @@ MTFLAG    MTFLAG    first
...
+# These are not relevant for exposures from multiple proposals
+#
+################################################################################
+<delete>  IPPPSSOO
+<delete>  ASN_ID
+<delete>  ASN_MTYP
+<delete>  ASN_TAB
+<delete>  GYROMODE
+<delete>  SUNANGLE
+<delete>  MOONANGLE
```
Comment (mdlpstsci) [inline drizzlepac/pars/acs_header_hap.rules:415]: Is this a typo?  MOONANGL vs MOONANGLE?

File: drizzlepac/pars/wfc3_header_hap.rules [hunk 3]
```diff
@@ -358,10 +358,25 @@ EQUINOX   EQUINOX   first
...
+# These are not relevant for exposures from multiple proposals
+#
+################################################################################
+<delete>  IPPPSSOO
+<delete>  ASN_ID
+<delete>  ASN_MTYP
+<delete>  ASN_TAB
+<delete>  GYROMODE
+<delete>  SUNANGLE
+<delete>  MOONANGLE
```
Comment (mdlpstsci) [inline drizzlepac/pars/wfc3_header_hap.rules:372]: Should this be MOONANGL?

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
There is an acs_mvm_header_hap.rules file.  Should there be a corresponding file for wfc3?

Just a few comments and questions to address.

File: drizzlepac/pars/wfc3_header_hap.rules (hunk 3 above)
Comment (stsci-hack) [reply drizzlepac/pars/wfc3_header_hap.rules]: Fixed in this file as well as 'wfc3_mvm_header_hap.rules' in a625671.

File: drizzlepac/pars/acs_header_hap.rules (hunk 2 above)
Comment (stsci-hack) [reply drizzlepac/pars/acs_header_hap.rules]: Fixed in this file along with 'acs_mvm_header_hap.rules' in a625671.

File: drizzlepac/haputils/processing_utils.py (hunk 1 above)
Comment (stsci-hack) [reply drizzlepac/haputils/processing_utils.py]: Fixed in a625671.

**Review by mdlpstsci — DISMISSED** [review]
Questions have been addressed via Slack and typos have been addressed -- good to go.

**Pull request merged**